e2e/basic.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {test, expect} from '@playwright/test';
+
+[
+  ['iframe', 'vanilla'],
+  ['iframe', 'preact'],
+  ['iframe', 'svelte'],
+  ['iframe', 'vue'],
+  ['iframe', 'htm'],
+].forEach(([sandbox, example]) => {
```

Comment:
there's no test.each in playwright: https://playwright.dev/docs/test-parameterize#parameterized-tests